tag.hbs

```diff
@@ -5,7 +5,7 @@
         {{#tag}}
             <section class="tag">
                 <div class="container medium">
-                    <h1 class="tag-name">{{name}}</h1>
+                    <h1 style="margin-bottom: 0;color: var(--brand-color);font-size: 2.4rem;">{{name}}</h1>
```

Comment:
Preferably I'd like to retain the `tag-name` class styling rather than shifting these properties into an inline style. Perhaps you could try removing this instead, think it should work!

![image](https://user-images.githubusercontent.com/2070423/152083028-18b3686c-763d-4dc7-bcf9-c4f8a1735742.png)